NativeApp/src/components/Home/container.js

```diff
@@ -67,6 +68,8 @@ export default Container => class extends Component {
       const { calendarDate } = this.state;
       getMonthEvents(calendarDate)
         .then(data => this.setState({ events: this.formatDate(data) }));
+      getUserEvents()
+        .then(res => this.setState({ upcomingEvents: res }));
```

Comment:
could make this... `.then(upcomingEvents => this.setState({ upcomingEvents }));` - but no big